eth2near/eth2near-block-relay-rs/src/eth2near_relay.rs

```diff
@@ -361,7 +378,7 @@ impl Eth2NearRelay {
         let end_period = BeaconRPCClient::get_period_for_slot(last_finalized_slot_on_eth);
         info!(target: "relay", "Last finalized slot/period on ethereum={}/{}", last_finalized_slot_on_eth, end_period);
 
-        if last_finalized_slot_on_eth - last_finalized_slot_on_near > 500 {
+        if last_finalized_slot_on_eth - last_finalized_slot_on_near >= 500 {
```

Comment:
Use a constant, please

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/775/commits/743ba13ebd6711e09f166bb9211aa66edb0d6657